src/Altinn.Platform/Altinn.Platform.Authentication/Altinn.Platform.Authentication.Tests/Utils/AssertUtil.cs

```diff
@@ -0,0 +1,21 @@
+using Altinn.Platform.Authentication.Model;
+
+using Xunit;
+
+namespace Altinn.Platform.Authentication.Tests.Utils
+{
+    /// <summary>
+    /// Util class for assertion.
+    /// </summary>
+    public static class AssertUtil
```

Comment:
I'm not too fond of the term "util". Could we call it something else? MoreAsserts? IntrospectionAsserts? AdvancedAsserts?